target/linux/bcm53xx/patches-5.10/342-Meraki-MX6X-add-legacy-LED-bindings.patch

```diff
@@ -0,0 +1,207 @@
+From: Matthew Hagan <mnhagan88@gmail.com>
+Date: Wed, 17 Mar 2021 10:47:39 +0000
+Subject: bcm-nsp: revert MX64/MX65 LED definitions
+
+Revert to the legacy label bindings and add indicator LEDs to the MX64/
+MX65 device trees.
```

Comment:
Please describe **why** instead of **what**. Why do we need legacy `label` here?

Reply:
This patch has been removed